dkt/README.markdown

```diff
@@ -77,9 +77,18 @@ cd
 rm -rf "$dktdir"
 ```
 
-If you have docket inside your `$GOPATH/src` tree, you can run it on the fly
-using `go run`:
+### Keeping in sync with your version of docket
 
-```sh
-go run github.com/bloomberg/docket/dkt
-```
+The `dkt` program in this directory doesn't actually contain much
+docket-specific logic. Instead, it forwards its arguments to the program in
+`dkt/main`, which actually makes calls to docket's packages.
```

Comment:
This is minor but i think it may be a little more clear to just describe what `dkt` does, rather than make an assumption about what the reader is assuming it does. So i'd considering changing this first graph to just:

The `dkt` program in this directory forwards its arguments to the program in `dkt/main`, which makes calls to docket's packages

Reply:
Applied your suggestion -- thanks!